### DISCUSSION TITLE
PacBio sequencing and Transposons

### DISCUSSION THREAD

**Discussion by Ramkyeri** [GuyTeichman/RNAlysis#18] [General]
Dear GuyTecichman

Thank you so much for develpeing this tool. it is more easer firendly. 

But I coud not find the running time,   because I want to run more than 100 samples (Genome size  is 2.5 Gb). 

Can we use this tool  to analysis long reds (PacBio sequencing) and Transposons from plant genome. 

with regards

**Comment by GuyTeichman** [comment]
Hi Ramekyeri,
Thank you for writing! I'm glad to hear you found this tool useful. 

I'll assume that you meant long-read RNA sequencing (as opposed to genomic DNA). If that's the case, you should be able to quantify expression in your PacBio sequencing samples using _kallisto_ (which is supported by RNAlysis), and all downstream analyses should be applicable to plant genomes as well. 

Good luck with your analysis,
Guy.

**Comment by Ramkyeri** [comment]
Dear GuyTecichman, 

Many thanks for your kind reply.  You mean that RNAlysis can analysis the gene expression from both illumina and pacbio sequencing. 

RNAlysis is a stand alone tool and can run on winodow, but  could you update the running time and system requirement. 

Can we apply this tool to Nanopore sequencing too?

with regards

**Comment by GuyTeichman** [comment]
Hi Ramekyeri,  
my apologies for the late response, I did not get a notification for your response. 
RNAlysis can run on any modern oprerating system (Windows >=10, MacOS>=11, Linux). Running times highly depend on the size of your dataset, type of analysis you perform, and the capability of your computer, but in theroy is should run in reasonable time on a personal computer or laptop. 

In principle, the tool can apply to Nanopore sequencing as well. 

Best,
Guy.
